uwblueprint/starter-code-v2
# Add Simple Entity Python API

## Notion ticket link

N/A



## Implementation description
* Created copies of existing entity API and frontend, without file storage
* Python version of this PR https://github.com/uwblueprint/starter-code-v2/pull/168



## Steps to test
1. Use the scrubber locally (yarn dev -t in starter-code-cli) with a Python backend (and any database).
2. Login on http://localhost:3000/ and verify that the new Create Simple Entity, Update Simple Entity, and Display Simple Entities pages work as expected.
3. Verify that the existing entity API is unaffected.



## What should reviewers focus on?
* 


## Checklist
